client/src/components/Home/Home.js

```diff
@@ -9,6 +11,10 @@ const Home = () => {
         way than on other rating forms.
       </p>
       <p>Have fun!</p>
+      <p>Don't have an account yet?</p>
+      <Link to="/register">
+        <ButtonBlue message="Register now!" />
+      </Link>
```

Comment:
This is **_not_** good semantics having a `button` inside an HTML anchor element which is essentially what `Link` will render to. Please change this.